src/identify_client.cpp

```diff
@@ -231,7 +231,9 @@ namespace {
 		, {"UM", "uTorrent Mac"}
 		, {"UT", "uTorrent"}
 		, {"VG", "Vagaa"}
+		, {"WD", "WebTorrent Desktop"}
 		, {"WT", "BitLet"}
+		, {"WW", "WebTorrent"}
```

Comment:
I don't think it would make a big difference, since Webtorrent peers can't connect to `RC_1_2`, only the desktop version can and I don't think it's used a lot for standard torrents anyway.

Reply:
makes sense